contracts/JBController.sol

```diff
@@ -29,6 +30,13 @@ import './abstract/JBTerminalUtility.sol';
 import '@openzeppelin/contracts/access/Ownable.sol';
 import '@openzeppelin/contracts/security/ReentrancyGuard.sol';
 
+/**
+@dev Custom Errors to replace the require statement and save gas
+*/
+error BAD_RESERVED_RATE();
+error BAD_REDEMPTION_RATE();
+error BAD_BALLOT_REDEMPTION_RATE();
```

Comment:
These are the "BAD" errors I was talking about in last review. I believe they should be prefixed with INVALID instead.

Reply:
Oh, I might have missed this let me update